lib/tasks/meetup-seed.rake

```diff
@@ -0,0 +1,126 @@
+require 'json'
+require 'open-uri'
+require 'date'
+require 'time'
+
+namespace :offerSeeds do
+  desc "Seeding current events from Event Collector API"
+  task meetup: :environment do
+
+    # starting seed API logic
+    puts 'start of API seed'
+    puts ""
+
+    # Array to catch all the tags that were unsuccessfully seeded.
+    error_array = []
+
+    Tag.all.each do |tag|
+      puts ""
+      puts "Accessing Event Collector API for tag #{tag.name}"
+      # Lat & long = center of Germany and 500km radius around.
+      event_api_url = "https://event-collector-api.herokuapp.com/meetupql?activity=#{tag.name}&lat=51.165691&lon=10.451526&radius=500"
+
+      # If response is too large, seed skips to next tag and logs unsuccessful tag in error array.
+      begin
+        response = URI.open(event_api_url).read
+      rescue OpenURI::HTTPError => e
+        puts "Error message: #{e.message}"
+        error_array << tag.name
+        puts "Logged tag in error array."
+        puts "Skipping to next tag"
+        next
+      else
+        puts "API URL successfully opened"
+        puts ""
+      end
+
+      data = JSON.parse(response)
+
+      puts "Creating new offer for each result of the tag #{tag.name}"
+      puts "#{data["Info"]["EventCount"]} offers will be created for #{tag.name}"
+
+      sleep(3)
+
+      data["Data"].each do |data_entry|
```

Comment:
It would be a good idea to limit the number offers we save to our DB, since each tag seems to be returning hundreds of events. I don't think we need that many.

Reply:
Yes, good idea - I will limit it to 100 per category for now.